model-server-lib/src/main/kotlin/org/modelix/model/server/light/LightModelServer.kt

```diff
@@ -54,14 +56,26 @@ import java.time.Duration
 import java.util.*
 import kotlin.time.Duration.Companion.seconds
 
-class LightModelServer(val port: Int, val rootNode: INode, val ignoredRoles: Set<IRole> = emptySet()) {
+class LightModelServer(val port: Int, val rootNode: INode, val ignoredRoles: Set<IRole> = emptySet(), additionalHealthChecks: List<IHealthCheck>) {
+    constructor(port: Int, rootNode: INode, ignoredRoles: Set<IRole> = emptySet()) : this(port, rootNode, ignoredRoles, emptyList())
```

Comment:
```suggestion
class LightModelServer(
    val port: Int,
    val rootNode: INode,
    val ignoredRoles: Set<IRole> = emptySet(),
    additionalHealthChecks: List<IHealthCheck> = emptyList()
) {
```
A secondary constructor seems quite verbose to me in this case.

Reply:
The reason was to avoid a breaking change in MPS, but `@JvmOverloads` is probably a better alternative.